syntax/status.sublime-settings

```diff
@@ -0,0 +1,17 @@
+{
+	"always_show_minimap_viewport": false,
+	"auto_indent": false,
+	"detect_indentation": false,
+	"draw_centered": true,
```

Comment:
How come you made only the status dashboard is centered?

Reply:
I saw the commit status panel rendering everything centered, which looked a bit strange. As I did not yet use each of the panels provided, by GitSavvy, I decided to remove it from each of them. 

We may add it to others, too.

Same with word_wrap. I know word_wrap might be a good idea for commit messages, but should kept disabled in other dashboards to avoid ugly line breaks.

Reply:
I don't like the centered position.

> I saw the commit status panel rendering everything centered

Where? Because we are not setting that property anywhere in GitSavvy.

Reply:
The question was: Why **only** in status? I created the settings file for status panel with "draw_centered" true first and copied the settings to all other files, when I realized the "show_commit" output panel rendering everything centered looks ugly and misplaced.